pkg/controller/storagecluster/stork.go

```diff
@@ -1117,6 +1121,12 @@ func getStorkSchedDeploymentSpec(
 		deployment.Spec.Template.Spec.TopologySpreadConstraints = topologySpreadConstraints
 	}
 
+	// If resources is specified in the spec, the resources specified by annotation (such as portworx.io/stork-cpu)
```

Comment:
there are 2 annotations stork-cpu and stork-scheduler-cpu, they might be different, after the spec change we unify them into one?

Reply:
Yes, in storagecluster spec we can't configure them separately